virt/logging_events_monitoring/virt-collecting-virt-data.adoc

```diff
@@ -1,25 +1,54 @@
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. The Alertmanager is also responsible for sending the alerts to external notification systems.
 
-include::modules/virt-about-collecting-virt-data.adoc[leveloffset=+1]
+include::modules/virt-collecting-data-about-your-environment.adoc[leveloffset=+1]
 
-include::modules/gathering-data-specific-features.adoc[leveloffset=+1]
+[id="additional-resources_collecting-data-about-your-environment"]
+[role="_additional-resources"]
+=== Additional resources
 
-include::modules/virt-must-gather-usage-targeted-vm-data.adoc[leveloffset=+1]
+* Configuring the xref:../../monitoring/configuring-the-monitoring-stack.adoc#modifying-retention-time-for-prometheus-metrics-data_configuring-the-monitoring-stack[retention time] for Prometheus metrics data.
+* Configuring the Alertmanager to send xref:../../monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[alert notifications] to external systems.
+* Collecting `must-gather` data for xref:../../support/gathering-cluster-data.adoc#support_gathering_data_gathering-cluster-data[{product-title}].
+* Collecting `must-gather` data for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/troubleshooting_openshift_data_foundation/index#downloading-log-files-and-diagnostic-information_rhodf[{rh-storage-first}].
+* Collecting `must-gather` data for xref:../../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#gathering-data-specific-features_virt-collecting-virt-data[{VirtProductName}].
+* Collecting Prometheus metrics for xref:../../monitoring/managing-metrics.adoc#querying-metrics-for-all-projects-as-an-administrator_managing-metrics[all projects] as a cluster administrator.
+
+include::modules/virt-collecting-data-about-vms.adoc[leveloffset=+1]
+
+[id="additional-resources_collecting-data-about-vms"]
+[role="_additional-resources"]
+=== Additional resources
+
+* Installing xref:../../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-existing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on Windows VMs.
+* Downloading and installing link:https://access.redhat.com/solutions/6957701[VirtIO drivers] on Windows VMs without host access.
+* Connecting to Windows VMs with RDP using the xref:../../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-rdp-console-web_virt-accessing-vm-consoles[web console] or the xref:../../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-rdp-console_virt-accessing-vm-consoles[command line].
+* Collecting `must-gather` data about xref:../../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#virt-must-gather-usage-targeted-vm-data_virt-collecting-virt-data[virtual machines].
+// * Collecting virtual machine memory dumps. [link TBD. CNV-20256]
+
+[id="must-gather-tool"]
+== must-gather tool
```

Comment:
Optional: This heading seems a little too fragment-ish when compared to the other subsection headings. I'd suggest changing this to "Using the must-gather tool" or something similar